q2_feature_classifier/_consensus_assignment.py

```diff
@@ -242,3 +243,9 @@ def _compute_consensus_annotation(annotations, min_consensus,
         consensus_fraction_result = 1.0
 
     return annotation, consensus_fraction_result
+
+
+def _annotate_method(taxa, method):
+    taxa = taxa.view(pd.DataFrame)
+    taxa['method'] = method
```

Comment:
```suggestion
    taxa['Method'] = method
```

![image](https://user-images.githubusercontent.com/274668/61891591-d6aa1200-aebe-11e9-9fa4-0fe80c3e29dc.png)

I am being a bit opinionated here...

Reply:
🍅 

lol fine